README.md

```diff
@@ -1 +1,53 @@
 # SignhostClientLibrary.Polly
+[![Nuget package](https://img.shields.io/nuget/v/EntrustSignhostClientLibrary.svg)](https://www.nuget.org/Packages/EntrustSignhostClientLibrary)
+
+This repository contains the implementation for the Signhost API Retry Client. It provides functionality for interacting with the Signhost API, with automatic retrying on failures using a custom retry policy.
+
+### Install
+Get it on NuGet:
+
+`PM> Install-Package SignhostClientLibrary.Polly`
+
+### Example code
+
+```c#
+var settings = new  SignHostApiClientSettings("AppName appkey", "apikey or usertoken");
+
+var retryClient = new SignHostApiRetryClient(settings);
+
+var transaction = await retryClient.CreateTransactionAsync(new Transaction
+{
+	Signers = new List<Signer>
+	{
+		new Signer
+		{
```

Comment:
Same here, can simply ref the main repo readme, and keep this example down to how to init the retry client.

Reply:
Done